volatility3/framework/plugins/linux/pidhashtable.py

```diff
@@ -236,8 +235,8 @@ def _generator(
         self, decorate_comm: bool = False
     ) -> interfaces.objects.ObjectInterface:
         for task in self.get_tasks():
-            offset, pid, tid, ppid, name = pslist.PsList.get_task_fields(
-                task, decorate_comm
+            offset, pid, tid, ppid, name, _creation_time = (
```

Comment:
Really appreciate you naming the unused vars, think it'll make it easier if/when we come back to make use of it in the future.  5:)